### ISSUE TITLE
Quick Start v2: VoiceOver: the completed cells state is not communicated

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12128]
Steps to reproduce:

1. Open a site in the app which has Quick Start v2 enabled or create a new site and select the option for quick start at the end of Site Creation ([screenshot](https://user-images.githubusercontent.com/1119271/61245040-d95f7700-a708-11e9-84ea-7ad966003389.png)).
1. Double tap "Completed (x)" to toggle it.
1. Double tap any item that is completed.

**Result:** the completed cells state is not communicated by VoiceOver. ([5s](https://cloudup.com/cEG86DqCTvT))

<img src="https://user-images.githubusercontent.com/1119271/61245369-8b973e80-a709-11e9-88a9-a3248cea691e.png" width="270" alt="quick-start-completed-cell">
<sup>Tested with WP Internal 12.8.0.201907011 on iPhone 6S iOS 12.3.1.</sup>

<p style="clear:both;"><sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub></p>

**Comment by guarani** [comment]
@designsimply Is the issue here that the VoiceOver commentary is missing an equivalent to the strikethrough - and therefore, should be something like the following?

> Create your site (completed)
> Get your site up and running

(Newcomer here, just poking around and came across this open issue.)

**Comment by etoledom** [comment]
Hey @guarani!  - That's right.

As you can see in the video from @designsimply , VoiceOver reads the completed cells as they were not completed. We need to tell VoiceOver to make VoiceOver communicate this state somehow.

**Comment by guarani** [comment]
Thanks for the confirmation, @etoledom! 

I can go ahead and look into appending "(completed)" to the accessibility label when quick start tasks are completed - the following part of `QuickStartChecklistCell.swift` looks relevant:
https://github.com/wordpress-mobile/WordPress-iOS/blob/372c9ed58261354207e65c5d04dc2cfce9e6e07b/WordPress/Classes/ViewRelated/Blog/QuickStartChecklistCell.swift#L26

Does that sound like a plan?

**Comment by etoledom** [comment]
Sounds good!
Also removing it when it moves to `not-completed` 👍

**Comment by guarani** [comment]
Sorry, I neglected this one 😔. I'll get to it after wrapping up my current PR.

**Comment by guarani** [comment]
> Sorry, I neglected this one 😔. I'll get to it after wrapping up my current PR.

I'm freed up now, so gonna tackle this in the coming days. (I hope I haven't held it back - if so, please don't hesitate to give it to someone with more bandwidth.)
